src/sv-pipeline/scripts/make_evidence_qc_table.py

```diff
@@ -27,6 +28,21 @@ def read_ploidy(filename: str) -> pd.DataFrame:
     return df_ploidy
 
 
+def read_sex_assignments(filename: str) -> pd.DataFrame:
+    """
+    Args:
+        filename: A tab-delimited file containing estimated copy numbers.
+    Returns:
+        A pandas DataFrame containing the following columns:
+        [id, chr1_CopyNumber, ..., chr22_CopyNumber, chrX_CopyNumber, chrY_CopyNumber, chrX_CopyNumber_rounded].
```

Comment:
I assume this documentation was a copy/paste error and should be updated to refer to the sex assignment data?